hist

```diff
@@ -1,29 +1,38 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
 VERSION="0.0.2"
-HISTORIAN_SRC=${HISTORIAN_SRC-"${HOME}/.bash_history"}
+HISTORIAN_BASH=${HISTORIAN_BASH-"${HOME}/.bash_history"} \
+HISTORIAN_ZSH=${HISTORIAN_ZSH-"${HOME}/.zsh_history"} \
 HISTORIAN_DB=${HISTORIAN_DB-"${HOME}/.historian.db"}
 HISTORIAN_SQLITE3=${HISTORIAN_SQLITE3-"$(which sqlite3)"}
-MAGIC="$(echo -e "\x10\x83\xB9\x9F\x34\xB5\x96\x45")" # 0118 999 881 999 119 725 3
```

Comment:
There was a semi-good reason for using a magic delimiter here, because it's quite hard to get things in the right format for normal SQLite import, so using a magic constant here would almost guarantee lines would never get split up.